views/scss/inc/_connectivity.scss

```diff
@@ -4,8 +4,9 @@
     margin-left: 40px;
     .icon-connect, .icon-disconnect {
         display: none;
-        line-height: 1.6;
-        @include font-size(23);
+        line-height: 2;
```

Comment:
I agree that this line-height is the best placement for the icon, but, 1.6 is more in line with the other things such as the title and progress bar.
<img width="1411" alt="screen shot 2017-08-27 at 10 55 13 pm" src="https://user-images.githubusercontent.com/2172596/29753921-679483ae-8b7b-11e7-8e97-e68af2f121c4.png">

Reply:
I understand your concern but I would not want to change this. Actually the progress bar has been diplaced at some point and now hangs too low. I suggest you take the PR as it is and I file a bug concerning the progress bar. How about it?